src/lib/components/CountdownWidget.svelte

```diff
@@ -18,7 +18,7 @@
 	});
 </script>
 
-<div class="flex flex-row gap-4 {className}">
+<div class="hidden sm:contents gap-4 {className}">
 	<div class="flex flex-row w-full tagWidget tagWidget-main grow pr-5">
```

Comment:
You can edit class names in two places rather than copy/paste everything

For example here:

```suggestion
	<div class="flex flex-col sm:flex-row w-full tagWidget tagWidget-main grow pr-5">
```

Same principle with the `justify-end` (`justify-end sm:justify-normal`)

This PR can be a 2-lines change